Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -117,7 +117,7 @@ The energy consumption should include all energy consumed by hardware reserved o
 
 #### Location-based marginal carbon intensity
 
-The carbon intensity of electricity is a measure of how much carbon (CO2eq) emissions are produced per kilowatt-hour (kWh) of electricity consumed. Because this specification uses a consequential approach, marginal emissions rates shall be used for electricity consumption. 
+The carbon intensity of electricity is a measure of how much carbon (CO2eq) emissions are produced per kilowatt-hour (kWh) of electricity consumed. Because this specification uses a consequential approach, marginal emissions rates shall be used for electricity consumption. The marginal emissions employed should consider short-run and long run effects when evaluating software.
```

Comment:
```suggestion
The carbon intensity of electricity measures how much carbon (CO2eq) emissions are produced per kilowatt-hour (kWh) of electricity consumed. Because this specification uses a consequential approach, marginal emissions rates shall be used for electricity consumption. The marginal emissions employed should consider short-run and long-run effects when evaluating software.
```

Reply:
Minor suggestions to crispen the language.

Reply:
I think that a discussion of the different signals (short run and long run) and potential proxies would be great in the guidance document. This could include potential sources as well as a discussion of relevant signals.

Reply:
Cool! Ok that sounds good to have it in the discussion document. If you want to go ahead and accept the "suggested change" via the "Commit suggestion" button, in the meantime, I can approve the PR (conditional of course on SWG acceptance tomorrow).